### PR TITLE
zh-CN: lite/convert/python_api.md

### PR DESCRIPTION
**Pull request by waenwj**

### CODE REVIEW

File: site/zh-cn/lite/convert/python_api.md [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+注意 : 本文介绍的是 Tensorflow nightly 版本的转换器， 运行 `pip install tf-nightly` 安装此版本。
+旧版文档请参考[“转换 TensorFlow 1.12 及之前版本的模型”](#pre_tensorflow_1.12)。
+
+[TOC]
+
+  
+## 概述
+
+虽然也可以在命令行中调用 TensorFlow Lite 转换器，但用 Python 脚本调用 API 的方式可以作为模型开发流水线 (model development pipeline)的一环，通常会更加便捷，
+它可以让您尽快知道您正在设计的模型能否可以针对移动设备。
```
Comment (gaoljhy) [inline site/zh-cn/lite/convert/python_api.md]: 没有换行

Comment (gaoljhy) [reply site/zh-cn/lite/convert/python_api.md]: 它指代的是该文档
该文档可以让你更早的了解正在设计的模型是否针对移动设备

File: site/zh-cn/lite/convert/python_api.md [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+`TFLiteConverter.from_keras_model_file()` 用于 `tf.Keras` 文件。
+[基本示例](#basic) 展示简单浮点模型的用法。[复杂示例](#complex) 展示更复杂的模型用法。
+
+## 基本示例 <a name="basic"></a>
+
+以下部分显示了如何把基本浮点模型从各种原始数据格式转换成 TensorFlow Lite FlatBuffers。
+
+### 使用 tf.Session 转换 GraphDef <a name="basic_graphdef_sess"></a>
+
+以下示例展示了如何使用 `tf.Session` 对象把 TensorFlow GraphDef 转换成 TensorFlow Lite FlatBuffer。
```
Comment (gaoljhy) [inline site/zh-cn/lite/convert/python_api.md]: 原文
The following example shows how to convert a TensorFlow GraphDef into a TensorFlow Lite FlatBuffer from a `tf.Session` object
建议修改为
以下示例展示了如何转换一个 TensorFlow GraphDef 到一个来自 `tf.Session` 对象的 TensorFlow Lite FlatBuffer

File: site/zh-cn/lite/convert/python_api.md [hunk 3]
```diff
@@ -0,0 +1,231 @@
...
+out = tf.identity(val, name="out")
+
+with tf.Session() as sess:
+  sess.run(tf.global_variables_initializer())
+  converter = tf.lite.TFLiteConverter.from_session(sess, [img], [out])
+  tflite_model = converter.convert()
+  open("converted_model.tflite", "wb").write(tflite_model)
+```
+
+### 使用文件转换 GraphDef <a name="basic_graphdef_file"></a>
```
Comment (gaoljhy) [inline site/zh-cn/lite/convert/python_api.md]: 原文
Exporting a GraphDef from file
建议修改为
从文件导出一个 GraphDef

File: site/zh-cn/lite/convert/python_api.md [hunk 4]
```diff
@@ -0,0 +1,231 @@
...
+
+以下示例展示了当 GraphDef 被存成文件时，如何转换成 TensorFlow Lite FlatBuffer，文件格式支持 `.pb` 和 `.pbtxt`。
```
Comment (gaoljhy) [inline site/zh-cn/lite/convert/python_api.md]: 原文
The following example shows how to convert a TensorFlow GraphDef into a TensorFlow Lite FlatBuffer when the GraphDef is stored in a file. Both `.pb` and `.pbtxt` files are accepted.
建议修改为
以下示例展示了当 GraphDef 被存成文件时，是怎样转换一个 TensorFlow GraphDef 到 TensorFlow Lite FlatBuffer。支持文件后缀为 `.pb` 和 `.pbtxt`。

File: site/zh-cn/lite/convert/python_api.md [hunk 5]
```diff
@@ -0,0 +1,231 @@
...
+input_arrays = ["input"]
+output_arrays = ["MobilenetV1/Predictions/Softmax"]
+
+converter = tf.lite.TFLiteConverter.from_frozen_graph(
+  graph_def_file, input_arrays, output_arrays)
+tflite_model = converter.convert()
+open("converted_model.tflite", "wb").write(tflite_model)
+```
+
+### 转换 SavedModel <a name="basic_savedmodel"></a>
```
Comment (gaoljhy) [inline site/zh-cn/lite/convert/python_api.md]: 原文 Exporting a SavedModel
以下 Exporting 都建议翻译为 导出 ，之后概不阐述

File: site/zh-cn/lite/convert/python_api.md [hunk 6]
```diff
@@ -0,0 +1,231 @@
...
+              optimizer=tf.keras.optimizers.RMSprop(lr=0.0001),
+              metrics=[tf.keras.metrics.categorical_accuracy],
+              sample_weight_mode='temporal')
+
+x = np.random.random((1, 3))
+y = np.random.random((1, 3, 3))
+model.train_on_batch(x, y)
+model.predict(x)
+
+# 将 tf.keras 模型保存成 HDF5 格式。
```
Comment (gaoljhy) [inline site/zh-cn/lite/convert/python_api.md]: 代码 默认不翻译
以下概不阐述

File: site/zh-cn/lite/convert/python_api.md [hunk 7]
```diff
@@ -0,0 +1,231 @@
...
+
+## 复杂示例 <a name="complex"></a>
+
+对于属性默认值不足的模型，在调用 `convert()` 之前应该设置属性值。
+例如设置任何常量都需要使用 `tf.lite.constants.<CONSTANT_NAME>`，以下示例中使用了常量 `QUANTIZED_UINT8`。
+您可以在 Python 终端中运行 `help(tf.lite.TFLiteConverter)` 获取有关属性的详细文档。
+
+尽管示例中只演示了包含常量的 GraphDefs，同样的逻辑可以应用于每一种输入数据格式。
+
+### 转换量化 GraphDef <a name="complex_quant"></a>
```
Comment (gaoljhy) [inline site/zh-cn/lite/convert/python_api.md]: 原文 Exporting a quantized GraphDef

建议修改为 导出一个量化的 GraphDef

File: site/zh-cn/lite/convert/python_api.md [hunk 8]
```diff
@@ -0,0 +1,231 @@
...
+### 源码构建 <a name="latest_package"></a>
+
+为了运行最新版本的 TensorFlow Lite Converter Python API，您可以选择一种方式安装 nightly 版本：
+[pip](https://www.tensorflow.org/install/pip)（推荐），
+[Docker](https://www.tensorflow.org/install/docker)，
+[从源代码构建 pip 包](https://www.tensorflow.org/install/source)。
+
+### 转换 TensorFlow 1.12 及之前版本的模型 <a name="pre_tensorflow_1.12"></a>
+
+参考下表，TensorFlow 1.12 及之前版本，将 TensorFlow 模型转换为 TensorFlow Lite。
```
Comment (gaoljhy) [inline site/zh-cn/lite/convert/python_api.md]: 原文
Reference the following table to convert TensorFlow models to TensorFlow Lite in
and before TensorFlow 1.12.

建议修改为
参考下表在 TensorFlow 1.12 之前的版本中转换 TensorFlow 模型到 TensorFlow Lite

File: site/zh-cn/lite/convert/python_api.md (hunk 3 above)
Comment (waenwj) [reply site/zh-cn/lite/convert/python_api.md]: Exporting 在之类表示表示 “导一个 GraphDef 变成 TensorFlow Lite FlatBuffer” ，所以没有直译 “导出”  。  

译成： “从文件导一个 GraphDef”

**Comment by waenwj** [comment]
Thanks  😄

File: site/zh-cn/lite/convert/python_api.md (hunk 1 above)
Comment (waenwj) [reply site/zh-cn/lite/convert/python_api.md]: “它” 指的是用”Python API“ 比”命令行“可以让你更早的了解正在设计的模型是否针对移动设备。

File: site/zh-cn/lite/convert/python_api.md (hunk 2 above)
Comment (waenwj) [reply site/zh-cn/lite/convert/python_api.md]: 调整成：
以下示例展示了如何从 `tf.Session` 对象转换一个 TensorFlow GraphDef 成 TensorFlow Lite FlatBuffer。
说明：
"from a `tf.Session` object" is for the whole sentence "how to convert a TensorFlow GraphDef into a TensorFlow Lite FlatBuffer".

File: site/zh-cn/lite/convert/python_api.md (hunk 1 above)
Comment (gaoljhy) [reply site/zh-cn/lite/convert/python_api.md]: 好的,理解偏差了[!😰]

File: site/zh-cn/lite/convert/python_api.md (hunk 2 above)
Comment (gaoljhy) [reply site/zh-cn/lite/convert/python_api.md]: 好的,收到

**Comment by gaoljhy** [comment]
lgtm

**Comment by gaoljhy** [comment]
@lamberta
